src/htsp_server.c

```diff
@@ -1083,13 +1083,34 @@ htsp_method_getSysTime(htsp_connection_t *htsp, htsmsg_t *in)
   htsmsg_t *out;
   struct timeval tv;
   struct timezone tz;
+  int tz_offset;
 
   if(gettimeofday(&tv, &tz) == -1)
     return htsp_error("Unable to get system time"); 
 
+  struct tm serverLocalTime;
+  tzset();
```

Comment:
Sorry for taking so long to respond @perexg . I've been away for a week.
I included this due to several internet references that say tzset should be called before localtime_r for portability. (E.g. https://mm.icann.org/pipermail/tz/2008-January/014789.html)